lib/seal.rb

```diff
@@ -24,14 +24,17 @@ def bark(mode: nil)
 
   def bark_at(team, mode: nil)
     message = case mode
-              when "quotes"
-                Message.new(team.quotes.sample) if team.quotes_days.map(&:downcase).include?(Date.today.strftime("%A").downcase)
+              when "morning_seal_quotes", "afternoon_seal_quotes"
+                send_quotes_message(team, mode)
               when "dependapanda"
-                MessageBuilder.new(team, :panda).build
+                if team.dependapanda
+                  MessageBuilder.new(team, :panda).build
+                  sleep 60
```

Comment:
I'd add an inline comment here, e.g.

```suggestion
                  sleep 60 # to prevent rate-limiting
```